ss/pebbledb/iterator.go

```diff
@@ -154,7 +153,7 @@ func (itr *iterator) Next() {
 		// append the current iterator key to the prefix and seek to that key.
 		itr.valid = itr.source.SeekLT(MVCCEncode(nextKey, itr.version+1))
 
-		tmpKey, _, ok := SplitMVCCKey(itr.source.Key())
+		tmpKey, tmpKeyVersion, ok := SplitMVCCKey(itr.source.Key())
```

Comment:
nit: this is critical code. I would add a godoc above stating what/why we're doing this.

Reply:
+1